src/app/infobox/infobox.component.ts

```diff
@@ -74,8 +75,26 @@ export class InfoboxComponent implements OnInit {
         window.speechSynthesis.cancel();
         window.speechSynthesis.resume();
         window.speechSynthesis.speak(msg);
-      }
+  }
+
+  Show() {
+    this.showModal = true;
+    let modal = document.getElementById("modo");
+    modal.style.display = "block"
+  }
+  close() {
+    this.showModal = false;
+    let modal = document.getElementById("modo");
+    modal.style.display = "none"
+  }
 
   ngOnInit() {
+    window.onclick = function(event) {
+      let modal = document.getElementById("modo");
+      if (event.target === modal) {
+        modal.style.display = "none"
```

Comment:
Same as above

Reply:
Yes we do. because for the modal to exit when we click out of the modal